### PR TITLE
updated VIP names in language files

### LINKED ISSUE #2636: Replace direct translations for VIP names with better localised alternatives
### What feature or enhancement would you like to add?

Our language files for Greek, Norwegian, Polish, Russian, Traditional Chinese and Simplified Chinese currently use direct translations for Theme Hospital's original VIP names. These could be replaced with more relevant and better localised alternatives, which can include national celebrities or politicians from those countries.

More pressingly, our Norwegian, Polish, Russian, Traditional Chinese and Simplified Chinese translations include the "Billy Savile OBE" reference, which should be replaced with our default substitute of "Sir Lancelot Spratt".

### How would this feature benefit the gameplay of CorsixTH?

Improved localisation will be appreciated by those who speak the above languages.

### Was the feature in the original game?

No

### What version of CorsixTH are you using?

0.68

### Does this feature/enhancement relate to a current problem?

No

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by MandrakeHorse**
*Partially fixes #2636*

This pull request replaces all references to "Billy Savile" with "Sir Lancelot Spratt". Please note that I used Google Translate for the Russian and both Chinese translations, so these should ideally be reviewed before merging this pull request. Alternatively, these strings can be replaced with better localised alternatives at a later date.

### CODE REVIEW

**Comment by MandrakeHorse** [comment]
Please note that this pull request doesn't completely address #2636, so we should avoid automatically closing the issue upon merging.

**Comment by lewri** [comment]
Just delete the strings entirely, they will default back to English.

File: CorsixTH/Lua/languages/simplified_chinese.lua [hunk 1]
```diff
@@ -2098,7 +2098,7 @@ vip_names = {
   "来自荷兰的伯那德国王",
   "缅甸民主党领袖：昂山苏蒂",
   "克朗伯先生",
-  "比利先生",
+  "兰斯洛特·斯普拉特爵士",
```
Comment (lewri) [inline CorsixTH/Lua/languages/simplified_chinese.lua:2101]: Replace this block as below:

```lua
vip_names = {
  health_minister = "卫生部部长",
  [1] = "伟大的查普顿市长", -- the rest is better organized in an array.
  [2] = "南丁格尔",
  [3] = "来自荷兰的伯那德国王",
  [4] = "缅甸民主党领袖：昂山苏蒂",
  [5] = "克朗伯先生",
  -- [6] requires replacement (English uses 'Sir Lancelot Spratt')
  [7] = "克劳福议员",
  [8] = "罗尼",
  [9] = "一个超级联赛球星",
  [10] = "拉里普罗斯特",
}
```

File: CorsixTH/Lua/languages/norwegian.lua [hunk 2]
```diff
@@ -2698,7 +2698,7 @@ vip_names = {
   "Kong Bernard av Nederland",
   "Aung Sang Su Kyi, den Burmesisk Demokratiske Opposisjonslederen",
   "Sir Reginald Crumbly",
-  "Billy Savile OBE",
+  "Sir Lancelot Spratt",
```
Comment (lewri) [inline CorsixTH/Lua/languages/norwegian.lua:2701]: Replace as below

```lua
-- VIP names
vip_names = {
  health_minister = "Helseministeren",
  [1] = "Ordföreren i Greater Trumpton", -- the rest is better organized in an array.
  [2] = "Lawrence Nightingale",
  [3] = "Kong Bernard av Nederland",
  [4] = "Aung Sang Su Kyi, den Burmesisk Demokratiske Opposisjonslederen",
  [5] = "Sir Reginald Crumbly",
  -- [6] requires replacement (English uses 'Sir Lancelot Spratt')
  [7] = "Rådgiver Crawford Purves",
  [8] = "Rocket Ronnie Jepson",
  [9] = "En Fotballspiller i Eliteserien",
  [10] = "L. F. Probst, III",
}

**Review by lewri — CHANGES_REQUESTED** [review]
Delete the others

**Comment by lewri** [comment]
Also https://github.com/CorsixTH/CorsixTH/issues/2636#issuecomment-2377733555

**Review by lewri — APPROVED** [review]
NB: Merge this before 2635
NB2: Add Argamax to the changelog at some point.

File: CorsixTH/Lua/languages/simplified_chinese.lua (hunk 1 above)
Comment (MandrakeHorse) [reply CorsixTH/Lua/languages/simplified_chinese.lua]: This does mean there'll be a 10% chance of an untranslated string appearing in the fax machine message when playing in Simplified/Traditional Chinese, but perhaps it's better to show an untranslated string than a badly translated string.

Comment (lewri) [reply CorsixTH/Lua/languages/simplified_chinese.lua]: Lokpro will likely pick up on it soon enough if they're still watching for translations; shouldn't be a massive issue.

File: CorsixTH/Lua/languages/russian.lua [hunk 3]
```diff
@@ -2610,16 +2610,16 @@ humanoid_name_ends = {
...
-  [4] = "Онг Сян Су Куй, предводитель бирманской демократической оппозиции",
-  [5] = "Сэр Риджинальд Крамбли",
-  [6] = "Сэр Вилли Сэвил",
-  [7] = "Советник Кроуфорд Первс",
-  [8] = "«Ракета» Ронни Джепсон",
-  [9] = "Футболист премьер-лиги",
-  [10] = "Л. Ф. Пробст III",
+  [1]  = "Мэр Нижнего Магнитошахтинска",
+  [2]  = "Леонид Каневский",
+  [3]  = '«Всезнающий» Онотоле Вассерман',
```
Comment (lewri) [inline CorsixTH/Lua/languages/russian.lua:2615]: @Matroftt is there a reason why this isn't using `"` instead?

Comment (Matroftt) [reply CorsixTH/Lua/languages/russian.lua:2615]: Used " symbols in name at first, but then switched to », and forgot to revert. My bad

Comment (lewri) [reply CorsixTH/Lua/languages/russian.lua:2615]: Coolio, @MandrakeHorse you cool to update the quotation marks?

Comment (MandrakeHorse) [reply CorsixTH/Lua/languages/russian.lua]: Had a bit of trouble with Git, but the single quotes have been replaced with double quotes

**Review by lewri — APPROVED** [review]

**Pull request merged**